Lib/fontTools/varLib/merger.py

```diff
@@ -1015,15 +1030,17 @@ def buildVarDevTable(store_builder, master_values):
 
 @VariationMerger.merger(ot.CaretValue)
 def merge(merger, self, lst):
-	assert self.Format == 1
+	if self.Format != 1:
+		raise VarLibMergeError(f"CaretValue format {self.Format} unknown.")
 	self.Coordinate, DeviceTable = buildVarDevTable(merger.store_builder, [a.Coordinate for a in lst])
 	if DeviceTable:
 		self.Format = 3
 		self.DeviceTable = DeviceTable
 
 @VariationMerger.merger(ot.Anchor)
 def merge(merger, self, lst):
-	assert self.Format == 1
+	if self.Format != 1:
+		raise VarLibMergeError(f"Anchor format {self.Format} unknown.")
```

Comment:
it's not like it is unknown, more like the merger doesn't support the second type of anchors (using contour points to position)